### PR TITLE
[ticket/13617] Enforce column size limit for session_forum_id 

### PR DESCRIPTION
**Pull request by naderman**
https://tracker.phpbb.com/browse/PHPBB3-13617

Ascraeus version of https://github.com/phpbb/phpbb/pull/3443

### CODE REVIEW

File: phpBB/phpbb/session.php [hunk 1]
```diff
@@ -130,6 +130,9 @@ static function extract_current_page($root_path)
 		$script_path .= (substr($script_path, -1, 1) == '/') ? '' : '/';
 		$root_script_path .= (substr($root_script_path, -1, 1) == '/') ? '' : '/';
 
+		$forum_id = request_var('f', 0);
```
Comment (MGaetan89) [inline phpBB/phpbb/session.php]: `$request->variable()`?

File: phpBB/phpbb/session.php [hunk 2]
```diff
@@ -130,6 +130,9 @@ static function extract_current_page($root_path)
...
+		$forum_id = ($forum_id > 0 && $forum_id < 16777215) ? $forum_id : 0;
```
Comment (MGaetan89) [inline phpBB/phpbb/session.php]: Comment about `16777215`?

File: phpBB/phpbb/session.php (hunk 1 above)
Comment (naderman) [inline phpBB/phpbb/session.php]: Indeed could change this, just kept request_var cause that was used before.

**Comment by nickvergessen** [comment]
Should we have a function for this, maybe even on the request class?

**Comment by marc1706** [comment]
I guess this is out of the scope of this PR though.

**Comment by naderman** [comment]
Indeed, might be useful but let's open up a new ticket for that, and also to review if there are more cases like this.

**Comment by marc1706** [comment]
For some reason the merge doesn't fast forward when merging the 3.0.x branch into the 3.1.x branch after having merged the different PRs for 3.0 and 3.1.

**Pull request merged**